content/docs/latest/installing/bare-metal/raspberry-pi.md

```diff
@@ -103,9 +103,60 @@ chmod +x flatcar-install
 mv flatcar-install ~/.local/bin
 ```
 
+##### Set up Ignition config
+
+Once the `flatcar-install` script has been installed, we can begin to write a Butane configuration file that can then be transpiled to Ignition.
+
+[Why Butane](butane-transpiler)? The Butane configuration transpiler reads from an easier-to-write language (YAML), and parses it through to check for errors. This means that if you write a valid Butane configuration, and it successfully transpiles to JSON, you are guaranteed to have a valid Ignition configuration.
+
+To generate your Ignition config, first start with the following `butane-config.yml`:
+
+```yaml
+variant: flatcar
+version: 1.0.0
+passwd:
+  users:
+    - name: core
+      ssh_authorized_keys:
+        - <paste authorized SSH keys here>
+
+
+kernel_arguments:
+  should_exist:
+    - console=tty0 # Used to tell the Linux kernel to output console to its display adapter
```

Comment:
Either this or the `/oem/grub.cfg` - which of them works?

Reply:
And is there a reason you removed `flatcar.autologin usbcore.autosuspend=-1`? Otherwise these should stay.